modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/trait/TraitUtils.java

```diff
@@ -452,7 +453,7 @@ else if (!fillRecursive(outTraits, inTraits, result, combination, idx + 1))
      * @param keys The keys to create collation from.
      * @return New collation.
      */
-    public static RelCollation createCollation(List<Integer> keys) {
+    public static RelCollation createCollation(Collection<Integer> keys) {
```

Comment:
I don't think it's a good idea to change the type of the param to `Collection`. Collation is supposed to have some order and this is unclear which order will be in case we pass a `Set`, for instance